scoring.js

```diff
@@ -0,0 +1,36 @@
+const calculateScore = (player) => {
+    switch(player.position) {
+        case "QB":
+            const passingScore = player.stats.passing.yards/25 + player.stats.passing.touchdowns*6 - player.stats.passing.interceptions*3
+            const rushingScore = player.stats.rushing.yards/10 + player.stats.rushing.touchdowns*6 - player.stats.rushing.fumbles*3
```

Comment:
Dont forget DRY don't repeat yourself.  There is alot of repetition here.  You could make a calculate score for your
Passing, rushing, receiving and returns and then incorporate them as needed for each position.